NthKey/Model/PersistentStore.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  PersistentStore.swift
+//  PersistentStore
+//
+//  Created by Sergey Vinogradov on 08/03/2021.
+//  Copyright © 2021 Purple Dunes. Distributed under the MIT software
+//  license, see the accompanying file LICENSE.md
+
+import CoreData
+
+struct PersistentStore {
+    static var preview: PersistentStore = {
+        let result = PersistentStore(inMemory: true)
+        let viewContext = result.container.viewContext
+        // wallet
+        let wallet = WalletEntity(context: viewContext)
+        let codes = [
+            "bc1qlpsgumjm2dlcljqc96c38n6q74jtn88enkr3wrz0rtp9jp6war7s2h4lrs",
```

Comment:
Are these hardcoded address for previews? If so, please add a comment and also use testnet addresses.